src/content/NewsContent.tsx

```diff
@@ -16,6 +16,14 @@
 import { PostDescription } from '../theme/types';
 
 const news: PostDescription[] = [
+  {
+    title: 'January 2022 Newsletter',
```

Comment:
I suggest we now use news instead of newsletter as for the published articles. I prefer we only use newsletter for what we will send by email to avoid misunderstanding
```suggestion
    title: 'January 2022 News',
```